src/style.css

```diff
@@ -93,39 +156,146 @@ body {
   flex-direction: column;
 }
 
-.hero__body {
-  color: var(--text-color);
-  font-weight: 400;
-  line-height: 1.5em;
+/* Works Section Styles */
+.works__cards-list {
+  margin: 0;
+  padding: 7em 0;
+  list-style: none;
+  display: grid;
+  gap: 5.5em;
 }
 
-/* Social Media Links Styles */
-.hero__social-media-links__title {
+.works__project-card {
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  gap: 0.8em;
+  border-radius: 1em;
+  padding: 1em;
+  background-color: var(--white);
+  border: 1px solid var(--border-color);
+}
+
+.works__project-card__screenshot {
+  border-radius: 0.5em;
+}
+
+.works__project-card__title {
+  font-weight: 700;
+  font-size: var(--fs-800);
+}
+
+.works__project-card__summary {
+  display: flex;
+  gap: 0.8em;
+}
+
+.works__project-card__client {
+  color: var(--text-color);
   text-transform: uppercase;
-  color: var(--primary-color-300);
-  font-weight: 500;
+  font-size: 300;
+  font-weight: 600;
+  line-height: 1em;
+}
+
+.works__project-card__role,
+.works__project-card__year {
+  color: var(--gray-600);
+  font-size: 300;
+  font-weight: 600;
+  line-height: 1em;
+}
+
+.works__project-card__intro {
+  color: var(--text-color);
+  font-size: var(--fs-300);
   line-height: 1.5em;
-  padding-bottom: 0.8em;
 }
 
-.hero__social-media-links__icons {
+.works__project-card__tech-stack-list {
   list-style: none;
   padding: 0;
   margin: 0;
+  margin-bottom: 0.5em;
   display: flex;
-  gap: 1.25em;
+  gap: 0.5em;
 }
 
-/* Project Cards Styles */
-
-/* Social Media Links Styles */
+.works__project-card__tech-stack-item {
+  color: var(--primary-color-400);
+  background-color: var(--light-blue);
+  padding: 0.25em 0.8em;
+  border-radius: 0.5em;
+}
 
 /* About Me Styles */
+.about-me {
+  background-color: var(--white);
+  display: flex;
+  flex-direction: column;
+  align-items: start;
+  gap: 0.5em;
+  padding: 7em 1.5em;
+  border-top-right-radius: 4em;
+}
 
-/* Social Media Links Styles */
+.about-me__header {
+  color: var(--dark-blue);
+  font-weight: 700;
+  font-size: var(--fs-900);
+}
 
-/* Contact Form Styles */
+.about-me__summary {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  gap: 2em;
+  list-style: none;
+  padding: 2em 0;
+  margin: 0;
+}
 
-/* Social Media Links Styles */
+.about-me__summary__header {
+  display: flex;
+  justify-content: space-between;
+  padding: 1.75em 1.5em;
+  color: var(--black);
```

Comment:
- [ ] Hey @henokkhm 👋 , you have done very well implementing the project requirements but there is still some adjustment to make your project look like the Figma template design. I will suggest you adjust the margin left for the `class="about-me__summary__header expanded" ` as defined in the original design.

> Figma Template
<img width="398" alt="Screenshot 2023-04-28 at 18 32 10" src="https://user-images.githubusercontent.com/71140133/235216845-db03fe9d-4ec9-4a87-9ae1-e6dd6323cdbf.png">

> Your design
<img width="385" alt="Screenshot 2023-04-28 at 18 30 46" src="https://user-images.githubusercontent.com/71140133/235216889-eb40c296-3bdd-4ea0-bab8-0954a167a31b.png">

Reply:
@Yerimah @iambenkis  @stanleySimeon  Is there any tool that I can use to automatically compare the Figma design with my implementation, or do I have to look for these discrepancies manually?